js/blocks/controls/fetch-input.js

```diff
@@ -108,6 +96,13 @@ class FetchInput extends Component {
 					results.length,
 					'block-lab'
 				), results.length ), 'assertive' );
+
+
+				if ( null === this.state.selectedSuggestion && '' !== this.props.value ) {
+					this.setState( {
+						selectedSuggestion: 0
```

Comment:
If there's no selectedSuggestion, make the first one selected. This allows the user to just type the first few letters of a name and press Enter (no down arrow).

Reply:
Great idea!

![top-selection](https://user-images.githubusercontent.com/4063887/54510299-5d981e00-4912-11e9-92a0-2a6cf7d487c5.gif)